src/main/java/com/EighthLight/app/Ai.java

```diff
@@ -45,66 +44,70 @@ public int findBestMove(Map<Integer, Integer>scoredSpaces)  {
         return bestSpace;
     }
 
-    public int miniMax(ArrayList boardState, int depth, String playerWhoMadeLastMove) {
-        depth++;
+
+    public int miniMax(ArrayList boardState, String playerWhoMadeLastMove) {
         ArrayList newBoard = new ArrayList();
         newBoard.addAll(boardState);
         ArrayList<Integer> emptySpaces = findEmptySpaces(newBoard);
         String playerWhoIsMoving = playerWhoMadeLastMove.equals(aiSymbol) ? humanSymbol : aiSymbol;
         ArrayList<Integer> scores = new ArrayList<>();
 
-        if (playerWhoMadeLastMove.equals(humanSymbol) && (ourBoard.hasAPlayerWon(newBoard, playerWhoMadeLastMove))) {
-            return -10 + depth;
-        } else if (playerWhoMadeLastMove.equals(aiSymbol) && (ourBoard.hasAPlayerWon(newBoard, playerWhoMadeLastMove))) {
-            return 10 - depth;
+        if (ourBoard.hasAPlayerWon(newBoard, playerWhoIsMoving)) {
+            return -10;
+        } else if (ourBoard.hasAPlayerWon(newBoard, playerWhoMadeLastMove)) {
+            return 10;
         } else if (emptySpaces.size() == 0) {
             return 0;
         }
-        
+
+        addScoreForSpace(emptySpaces, newBoard, playerWhoIsMoving, scores);
```

Comment:
I'm not saying you have to change anything here, but just as something to think about: it's hard to tell at first glance that that one of the 4 arguments to this function is being altered. It's good to be aware of these sorts of semi-silent destructive operations, and think about how to make them as explicit as possible, or better yet non-destructive.